tests/dummy/app/controllers/horizontal-bar.js

```diff
@@ -41,19 +42,20 @@ export default SlideController.extend({
   }),
 
   rawDataHash: Ember.computed(function() {
-  	return {
-	    asset_values: asset_values,
-	    many_values: many_values,
-	    monthly_return_single_period: monthly_return_single_period,
-	    high_net_worth_duration: high_net_worth_duration,
-	    '----': data.null,
-	    empty: data.empty,
-	    one_value: one_value,
-	    two_values: two_values,
-	    zero: zero,
-	    zeroes: zeroes,
-	    sum_to_zero: sum_to_zero,
-	    bad_range: bad_range
+    return {
```

Comment:
This diff got mangled because I replaced indentation with spaces, but the only actual change here the new key at the end